lib/shared/sdk/usbboot/index.js

```diff
@@ -361,15 +349,12 @@ exports.scan = (options) => {
   return usb.listDevices().filter(isUsbBootCapableUSBDevice).map((device) => {
   /* eslint-enable lodash/prefer-lodash-method */
 
-    const idPair = _.join([
```

Comment:
Could we also keep this idPair and print it in the debug messages?

Reply:
Sure, fixed!